packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - [Bugfix] Pin `@chakra-ui/react` version to 2.7.0 to avoid breaking changes from 2.10.9 [#2658](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2658)
 - Introduce optional prop `hybridAuthEnabled` to control Hybrid Auth specific behaviors in commerce-sdk-react [#3151](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3151)
 - Inject sfdc_user_agent request header into all SCAPI requests for debugging and metrics prupose [#3183](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3183)
+- Fix config parsing to gracefully handle missing properties [#3230](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/3230)
```

Comment:
We probably want to add this same changelog entry to the pwa-kit-create-app changelog

Reply:
I decided _not_ to update the changelog for pwa-kit-create-app because the changes there are all related to shoppper agent, which has not been released yet. So we don't need to say we fix something that isn't released yet.